src/main/osd/osd_elements.c

```diff
@@ -923,7 +923,12 @@ static void osdElementLogStatus(osdElementParms_t *element)
         } else if (isBlackboxDeviceFull()) {
             tfp_sprintf(element->buff, "%c>", SYM_BBLOG);
         } else {
-            tfp_sprintf(element->buff, "%c%d", SYM_BBLOG, blackboxGetLogNumber());
+            int32_t logNumber = blackboxGetLogNumber();
+            if (logNumber >= 0) {
+                tfp_sprintf(element->buff, "%c%d", SYM_BBLOG, logNumber);
+            } else {
+                tfp_sprintf(element->buff, "%c ", SYM_BBLOG);
```

Comment:
Seems like there's an unnecessary whitespace after the symbol. But perhaps it would make more sense to display a "?" as an indicator of an unknown next log number.

Reply:
Yeah, I should remove the whitespace. I left it in while I was humming and harring about adding another symbol. Not sure that a question mark is right - this would probably be understood by users as 'status of blackbox logging is unknown'. Having just the symbol is probably best if the log file number cannot be displayed.